v2/srv_integration_test.go

```diff
@@ -1,11 +1,14 @@
 package dnsutils
 
 import (
-	"github.com/stretchr/testify/assert"
+	"context"
+	"errors"
 	"os"
 	"strconv"
 	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
This new import ordering was created automatically by `goimports`. Let me know if you'd like me to revert it.